resources.md

```diff
@@ -8,21 +8,115 @@ This is a list of useful information we have gathered throughout our research an
 
 Meta Mesh is not responsible for any of this material, unless noted.
 
-## Active Community Wireless Network Projects
+## Active Community Wireless Network Projects (No defunct or stagnated projects. This is updated on a best-effort basis. Emails to organizing entities are sent out monthly requesting updates.)
```

Comment:
Can you fix this as suggested [here](https://github.com/metamesh/metamesh.org/commit/5143a5e384b4df1a28755de38587a019fe2a2a71#commitcomment-11601671)?